docs/source/monitoring_stack.rst

```diff
@@ -338,6 +342,10 @@ If Scylla monitoring IP is 10.0.0.1, the file should look like
 
 Restart rsyslog for the configuration to take effect.
 
+.. code-block:: sh
+
+
+   systemctl restart rsyslog
```

Comment:
why restart and not start?

Reply:
See the first paragraph of this doc, it should be working already